compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -254,8 +255,22 @@ class ResolverImpl(
 
         val superDescriptor = resolveForOverride(overridee) as? CallableMemberDescriptor ?: return false
         val subDescriptor = resolveForOverride(overrider) as? CallableMemberDescriptor ?: return false
+        val subClassDescription = overrider.closestClassDeclaration()?.let {
+            resolveClassDeclaration(it)
+        }
+        val superClassDescription = overridee.closestClassDeclaration()?.let {
+            resolveClassDeclaration(it)
+        }
+        if (subClassDescription == null || superClassDescription == null) return false
+        val typeOverride = subClassDescription.getAllSuperClassifiers()
+            .filter { it != subClassDescription } // exclude subclass itself as it cannot override its own methods
+            .any {
+                it == superClassDescription
```

Comment:
Can these 2 lamdas be merged into one?

Reply:
`Sequence<ClassifierDescriptor>` returns a sequence so i kept them separate for readability (and does not have the cost of creating intermediate lists).
happy to change if you have a strong preference.